lms/envs/common.py

```diff
@@ -434,6 +434,17 @@
     'ENABLE_FOOTER_MOBILE_APP_LINKS': False,
 
     # Let students save and manage their annotations
+    # .. toggle_name: FEATURES['ENABLE_EDXNOTES']
+    # .. toggle_implementation: SettingToggle
+    # .. toggle_default: False
+    # .. toggle_description: This toggle enables the students to save and manage their annotations in the
+    #   course using the notes service. The bulk of the actual work in storing the notes is done by
+    #   by a separate service(see the edx-notes-api repo).
+    # .. toggle_warnings: Requires the edx-notes-api service properly running and to have configured the django settings
+    #   EDXNOTES_INTERNAL_API and EDXNOTES_PUBLIC_API
```

Comment:
Do you mind adding a warning to keep this in sync with Studio?
```suggestion
    #   EDXNOTES_INTERNAL_API and EDXNOTES_PUBLIC_API. If you update this setting, also update in Studio.
```
Additionally, you could replace three lines of comments in cms here: https://github.com/edx/edx-platform/blob/2d7a8358efe563b6e1968f56229e4f72c9740857/cms/envs/common.py#L228-L231 with something like the following:
```
# See LMS annotations for details.
```